types/index.d.ts

```diff
@@ -37,26 +37,37 @@ declare global {
         /**
          * Array representing the current selection plus any locked items that the user has attempted to select.
          */
-        itemsIncludingLocked: Array<SceneNode>;
+        readonly itemsIncludingLocked: Array<SceneNode>;
```

Comment:
Modifying this array will not modify the items. I'd suggest making the array itself Readonly as well. (This suggestion applies to the `.items` property, too.)

Marking the arrays as Readonly will potentially save people from making the mistake of adjusting the "selected nodes array" returned by these getters and think it will affect the actual selection internally (it won't).

```suggestion
        readonly itemsIncludingLocked: ReadonlyArray<SceneNode>;
```